provisioning_templates/provision/kickstart_default.erb

```diff
@@ -43,7 +43,16 @@ This template accepts the following parameters:
   chef_enabled = @host.respond_to?(:chef_proxy) && @host.chef_proxy
   section_end = (rhel_compatible && os_major <= 5) ? '' : '%end'
   use_ntp = host_param_true?('use-ntp') || (is_fedora && os_major < 16) || (rhel_compatible && os_major <= 7)
+  iface = @host.provision_interface
+  network_options = []
 %>
+
+<%- if iface.tag.present? && iface.attached_to.present?
+  network_options.push = "--device=#{iface.attached_to} --vlanid=#{iface.tag}"
+else
+  network_options.push = os_major >= 6 ? " --device=#{@host.mac}" : ''
```

Comment:
This and line 51 are syntax errors, `push` is a method which you call. Remove the equal sign.

```suggestion
  network_options.push(os_major >= 6 ? " --device=#{@host.mac}" : '')
```